openslides_backend/action/actions/meeting/create_update_delete.py

```diff
@@ -104,6 +115,169 @@
 ]
 
 
+class MeetingCreate(CreateActionWithDependencies):
+    dependencies = [
+        MotionWorkflowCreateSimpleWorkflowAction,
+        GroupCreate,
+        GroupCreate,
+        GroupCreate,
+        GroupCreate,
+        GroupCreate,
+        ProjectorCreateAction,
+    ]
+
+    def base_update_instance(self, instance: Dict[str, Any]) -> Dict[str, Any]:
+        instance = super().base_update_instance(instance)
+
+        admin_group_fqid_list = [
+            key
+            for key, data in self.datastore.additional_relation_models.items()
+            if key.collection.collection == "group" and data.get("name") == "Admin"
+        ]
+        if len(admin_group_fqid_list) != 1:
+            raise ActionException(
+                "There is no admin group created during meeting.create."
+            )
+        admin_group_fqid = admin_group_fqid_list[0]
```

Comment:
This seems hacky. Proposed solution: Create the group explicitely here by calling `execute_other_action`. Adjust `execute_other_action` to return the action result (see https://github.com/OpenSlides/openslides-backend/blob/0860863d63a1cdd78e8001392ab92dfe980ca5a3/openslides_backend/action/action.py#L408 where it is currently ignored). You can then use this result (which contains the newly created id) to add the user to this group.

Reply:
Reworked with execute_other_action with return values write_request and action_results